app/search_builders/curation_concerns/collection_search_builder.rb

```diff
@@ -1,33 +1,42 @@
 module CurationConcerns
+  # Our parent class is the generated SearchBuilder descending from Blacklight::SearchBuilder.
+  # Defines search_params_logic used when searching for Collections.
+  # @see https://github.com/projecthydra/curation_concerns/blob/master/app/search_builders/curation_concerns/README.md SearchBuilders README
+  #
   class CollectionSearchBuilder < ::SearchBuilder
-    # Defines which search_params_logic should be used when searching for Collections
-    self.default_processor_chain = [:default_solr_parameters, :add_query_to_solr,
-                                    :add_access_controls_to_solr_params, :add_collection_filter, :some_rows, :sort_by_title]
+    # @param [#blacklight_config, #current_ability] scope the object that has access to #blacklight_config
+    #   From a controller, scope would be `self`.  This argument passed to ::SearchBuilder.new()
+    # @param [Array{String}] permissions defining breadth of search, e.g. 'edit', 'read', 'discover'
+    # @note permissions will otherwise be defaulted by inherited #discovery_permissions
+    def initialize(scope, permissions = nil)
+      @rows = 100
+      @discovery_permissions ||= permissions if permissions && !permissions.empty?
+      super(scope)
+    end
+
+    # @return [String] class URI of the model, as indexed in Solr has_model_ssim field
+    def model_class_uri
+      ::Collection.to_class_uri
+    end
 
-    def some_rows(solr_parameters)
-      solr_parameters[:rows] = '100'
+    # @return [String] Solr field name indicating default sort order
+    def sort_field
+      'title_si'
     end
 
-    def add_collection_filter(solr_parameters)
+    # @!group Overrides
+
+    # This overrides the filter_models in FilterByType
+    def filter_models(solr_parameters)
```

Comment:
Yeah, I was tempted to do something similar.  Dislike the name `models` for the method though.